### PR TITLE
Feature/setup and controllers

### PR DESCRIPTION
**Pull request by mistermaxis**
### Tasks Completed:
- Add basic setup.
- Create controllers for users and posts.

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -2,5 +2,9 @@
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
 
   # Defines the root path route ("/")
-  # root "articles#index"
+  root "users#index"
+
+  resources :users do
+    resources :posts
+  end
```
Comment (Dipeshtwis) [inline config/routes.rb:9]: - [ ] I would recommend maintaining a RESTful URL, i.e. Do not expose unnecessary endpoints.

```ruby
# config/routes.rb

  resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end
```

File: README.md [hunk 2]
```diff
@@ -1,24 +1,51 @@
...
-* Configuration
+- Major languages
+- Frameworks
+- Technologies used
 
-* Database creation
+## Live Demo (if available)
 
-* Database initialization
+[Live Demo Link](https://livedemo.com)
```
Comment (Dipeshtwis) [inline README.md:16]: - [ ]  Let's fix the `Live Demo` section. Please remove that section for now if you don't have a live link to the project.

**Review by Dipeshtwis — CHANGES_REQUESTED** [review]
Hi @mistermaxis, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N.A.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by bimbolabuari — APPROVED** [review]
Hi @mistermaxis, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request closed without merging**